build.gradle

```diff
@@ -0,0 +1,87 @@
+subprojects {
+    apply plugin: 'java'
+    apply plugin: 'scala'
+
+    apply from: "${rootProject.projectDir}/gradle-scalatest.plugin"
+
+    ext {
```

Comment:
I agree in principle, but can't point to any specific line / part of line I'd move. Do you have specifics in mind?